charts/primary-site/templates/deployments/inbox-listener.yaml

```diff
@@ -18,6 +18,9 @@ spec:
     metadata:
       labels:
         app: inbox-listener
+        {{- with .Values.inboxListener.deployment.podLabels }}
```

Comment:
is `podLabels` the conventional term for this from other helm charts?

Reply:
yeah, this is the term they use in the bitnami charts for it